pkg/kubectl/categories/categories.go

```diff
@@ -129,12 +140,16 @@ func (e discoveryFilteredExpander) Expand(category string) ([]schema.GroupResour
 	return available, ok
 }
 
+// UnionCategoryExpander represents for a list of CategoryExpander,
+// expends to a union of gourpResources lists, which merges all expentions of CategoryExpander in UnionCategoryExpander.
 type UnionCategoryExpander []CategoryExpander
 
+//Expand returns a single list of all GroupResource objects. If not found, return empty slice and false
```

Comment:
This comment appears to be repeating information. I guess if we say that UnionCategoryExpander implements CategoryExpander interface, then we can skip documenting this method.

Reply:
done